src/test/frontend/db/reference_test.cljs

```diff
@@ -0,0 +1,182 @@
+(ns frontend.db.reference-test
+  (:require [cljs.test :refer [deftest is testing]]
+            [datascript.core :as d]
+            [logseq.db :as ldb]
+            [logseq.db.common.reference :as db-reference]
+            [shadow.resource :as rc]))
+
+(def test-transit (rc/inline "fixtures/references.transit"))
+;; (use-fixtures :each test-helper/db-based-start-and-destroy-db)
+
+(defn- create-conn!
+  []
+  (let [db (ldb/read-transit-str test-transit)]
+    (d/conn-from-db db)))
+
+;; FIXME: EDN import doesn't work
```

Comment:
I tried to import the blocks from my test graph, but the result doesn't look right.
So we went to export the test graph to a transit file.